ai/docs/Vicuna Model Deployment and Training Documentation.md

```diff
@@ -0,0 +1,125 @@
+### 3.Check if the model can repsonse
+
+```plain
+sh call.sh
+```
+It will send POST request to call the model, and return the model reponses. If the responses are like this, the model has been deployed successfully. 
+```plain
+{"id":"chatcmpl-UFWP9BqDg3s7DDfc5eXLS3","object":"chat.completion","created":1701329914,"model":"vicuna","choices":[{"index":0,"message":{"role":"assistant","content":"Sure! Here are three numbers between 1 and 10:\n\n1. 5\n2. 8\n3. 9"},"finish_reason":"stop"}],"usage":{"prompt_tokens":51,"total_tokens":81,"completion_tokens":30}} 
+```
+If the container has not been built, it may return this message, and you should wait for a while. If it consistently returns this message, there might be an issue:
+```plain
+{"object":"error","message":"Only  allowed now, your model vicuna","code":40301} 
+```
+### 4.Interactive with the model
+
+```plain
+sh interactive.sh
+```
+You can chat with the deployed Vicuna model by running this Shell script. Feel free to ask any questions interactively to the model. To exit, simply press Ctrl+Z. Please note that the JSON processing tool **jq** is required for this operation.
+
+## Fine-tuning
+
+### 1.Get the model and dataset
+For fine-tuning, you should set the Vicuna model and training dataset ready. Getting model shares the same steps as deployment, and training dataset should be collected in the form of json.
+One sample of the json dataset file is like this:
+```plain
+    {
+        "id": "1",
+        "conversations": [
+            {
+                "from": "human",
+                "value": "What is antctl and what are its different operating modes"
+            },
+            {
+                "from": "gpt",
+                "value": "Antctl is the command-line tool for Antrea, a significant component in the Kubernetes networking framework. It operates in three distinct modes: controller mode, agent mode, and standalone mode. In controller mode, it can connect to the Antrea Controller when run out-of-cluster or within the Controller Pod, enabling queries about network policies. In agent mode, when executed within an Antrea Agent Pod, it interacts with the local agent to provide information specific to that Agent, such as the set of computed Network Policies. The standalone mode is not explicitly described in the provided excerpt of the article."
+            }
+        ]
+    },
+```
+An example is like "./antrea/ai/vicuna/dataset/dummy_conversation.json". After the model and dataset being set ready, you should change to the "./antrea/ai/vicuna/train_full" directory for full parameter fine-tuning, or "./antrea/ai/vicuna/train_qlora" for QLoRA fine-tuning.
+
+### 2.Build container for training
```

Comment:
To avoid confusion with pretraining, using fine-tuning is more accurate and helps prevent misunderstandings. https://github.com/lm-sys/FastChat#fine-tuning

Reply:
Please also update the following content to make the description more accurate.